### PR TITLE
Fixes #15933: Make ncf.py usable by rudder-lang

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15933

### CODE REVIEW

File: tools/ncf.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 # Verbose output
 VERBOSE = 0
+CFENGINE_PATH="/opt/rudder/bin/cf-promisess"
```
Comment (amousset) [inline tools/ncf.py]: :snake: `s/ss/s/`

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
Merged with commit 7c0cae9660f41a581b0f77998481a2c24fdc4a02

**Pull request closed without merging**